src/libchdr_chd.c

```diff
@@ -319,6 +318,14 @@ static const UINT8 nullsha1[CHD_SHA1_BYTES] = { 0 };
     PROTOTYPES
 ***************************************************************************/
 
+/* core_file wrappers over stdio */
+static core_file *core_stdio_fopen(char const *path);
+static UINT64 core_stdio_fsize(core_file *file);
+static size_t core_stdio_fread(void *ptr, size_t size, size_t nmemb, core_file *file);
+static int core_stdio_fclose(core_file *file);
+static int core_stdio_fclose_nonowner(core_file *file); // alternate fclose used by chd_open_file
+static int core_stdio_fseek(struct chd_core_file* file, long offset, int whence);
```

Comment:
The first argument could be type `core_file*` instead of `struct chd_core_file*`.

Reply:
Okay, I fixed this on in the latest push.